stride/problem/problem.py

```diff
@@ -144,9 +168,23 @@ def time_resample(self, new_step, new_num=None, **kwargs):
         old_num = self.time.num
         self.grid.time.resample(new_step=new_step, new_num=new_num)
 
-        for shot in self.acquisitions.shots:
-            shot.wavelets = shot.wavelets._resample(factor=old_step/new_step, new_num=new_num)  # resample wavelet
-            shot.observed = shot.observed._resample(factor=old_step/new_step, new_num=new_num)  # resample observed
+        freq_niquist_hz = min(1/old_step, 1/new_step)  # anti-aliasing filter using max sampling frequency
```

Comment:
nyquist

Reply:
Should the calculation of the frequency and its conversion to dimensionless below be done in the `Traces._resample` method?